x/committee/keeper/grpc_query.go

```diff
@@ -0,0 +1,183 @@
+package keeper
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/kava-labs/kava/x/committee/types"
+)
+
+var _ types.QueryServer = QueryHandler{}
+
+type QueryHandler struct {
```

Comment:
testing with creating a new handler for grpc queries instead of using the keeper here to avoid potential func conflicts. thoughts? @nddeluca @rhuairahrighairidh

Reply:
+1 to avoiding name conflicts. Could also not embed the keeper to avoid exporting Keeper methods to users of the `QueryHandler`

Reply:
+1 to avoiding name conflicts. Could also not embed the keeper to avoid exporting Keeper methods to uses of the `QueryHandler`